### DISCUSSION TITLE
Zoom plugin disrupts deleting images

### DISCUSSION THREAD

**Discussion by sapkauskasm** [igordanchenko/yet-another-react-lightbox#37] [Q&A] [answered]
Let's say I add a "Delete" button to delete the current image in Lightbox.

On delete, I remove slide from slides array, and works as expected.

But...
If zoom plugin is present, the main image is not changed. 

Here is example:
https://codesandbox.io/s/yet-another-react-lightbox-examples-forked-j65dm5?file=/src/examples/PhotoGallery.js

On photo library, delete is working.
On "Failing photo library" after delete, image stys the same. Only difference is Zoom plugin.

**Comment by igordanchenko** [comment, marked as answer]
@sapkus this bug will be fixed in #38

**Reply by igordanchenko** [reply]
The fix is available in v2.2.2

**Reply by sapkauskasm** [reply]
Amazing!
